### PR TITLE
Add support for SCM repo link endpoints

### PR DESCRIPTION
**Pull request by EtienneM**
Related to Scalingo/dashboard-vue#271

### CODE REVIEW

File: src/SCMRepoLinks/index.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        .post(`/apps/${appID}/scm_repo_link`, { scm_repo_link: opts }),
+      'scm_repo_link',
+    )
+  }
+
+  /**
+   * Update a SCM integration link.
+   * @see https://developers.scalingo.com/scm_repo_link
+   * @param {String} appID - ID of the application
+   * @param {SCMRepoLinkCreateOpts} opts - SCM repo link information to update
```
Comment (johnsudaar) [inline src/SCMRepoLinks/index.js]: Those are not the same, some options are not available in update (like aut_integration_uuid, source)

File: src/SCMRepoLinks/index.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+  /**
+   * Manually deploy an application
+   * @see https://developers.scalingo.com/scm_repo_link#manual-deploy
+   * @param {String} appID ID of the application
+   * @param {String} branch Name of the branch to deploy.
+   * @return {Promise<Deployment, APIError>} Promise that when resolved returns the Deployment started.
+   */
+  manualDeploy(appID, branch) {
+    return unpackData(
```
Comment (johnsudaar) [inline src/SCMRepoLinks/index.js]: You should unpack the deployment.

Comment (EtienneM) [reply src/SCMRepoLinks/index.js]: I don't understand your comment

Comment (johnsudaar) [reply src/SCMRepoLinks/index.js]: The API will return a json looking like :

```
{
"deployment": {
   "id": "123456"
   ...
}
}
```

Here you will send the exact same object to the caller.
We want to send him (as the documentation say):

```
{
"id":"123456",
...
}
```

Comment (EtienneM) [reply src/SCMRepoLinks/index.js]: Woupsy

**Pull request merged**